index.html

```diff
@@ -53,150 +53,153 @@ <h1>I'm Benjamin Glad to see you!</h1>
               <a href="myself-btn"><img src="photo/logos/medium.jpg" alt=""></a>
             </div>
           </div>
-      </section>
-    
-    <section id="tul-1" class="middle-section-container">
+        </div>
+
+      <!-- All cards will start here with card1 -->
```

Comment:
- [ ] [OPTIONAL] You should create the project section dynamically, kindly remove this code (from line `60` to line `199`) from your HTML and create it using the JavaScript